backend/api/Database/Models/InspectionFindings.cs

```diff
@@ -1,22 +1,46 @@
-﻿using Microsoft.EntityFrameworkCore;
-
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Api.Controllers.Models;
 #pragma warning disable CS8618
 namespace Api.Database.Models
 {
-    [Owned]
     public class InspectionFindings
     {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public string Id { get; set; }
```

Comment:
In the Inspection models, I think we should have a reference to this such that an Inspection can contain a List<InspectionFindings>

Reply:
But inspection model already contains an inspectionFindingsList

Reply:
Agreed that we have them as list.